src/common/thrift/ThriftClientManager.inl

```diff
@@ -63,6 +63,11 @@ std::shared_ptr<ClientType> ThriftClientManager<ClientType>::client(
             delete p;
         });
     });
+
+    if (shared_) {
```

Comment:
Why remove the related host-evb pair only in share mode?

Reply:
Because in the shared mode, the clientMap_ object will be released after the eventbase object is released, so we need to delete the value corresponding to clientMap_ while the eventBase is released.

Reply:
I mean could we just delete the entry in clientMap_ whether in shared mode or not?  Because the clientMap_ is threadLocal,  it means it should be destroyed before related evb released.  WDYT?

Reply:
Yes, I am going to modify it.